### PR TITLE
feat: added quotes

### PR DESCRIPTION
**Pull request by TejasShekar**
- Added API for providing random quotes

[Preview Link](https://simpli-tab-git-quotes-tejasshekar.vercel.app/)

### CODE REVIEW

File: public/manifest.json [hunk 1]
```diff
@@ -1,25 +1,20 @@
...
-      "src": "logo192.png",
-      "type": "image/png",
-      "sizes": "192x192"
-    },
-    {
-      "src": "logo512.png",
-      "type": "image/png",
-      "sizes": "512x512"
-    }
+  "manifest_version": 2,
```
Comment (krgarima) [inline public/manifest.json:2]: You should change it to manifest version 3

File: src/App.css [hunk 2]
```diff
@@ -5,13 +5,27 @@ body,
...
   padding: 0;
   margin: 0;
-  color: white;
   font-size: 100%;
   line-height: normal;
+  color: white;
+}
+
+.screen {
+  background-color: rgba(0, 0, 0, 0.4);
```
Comment (krgarima) [inline src/App.css:14]: You forgot to use CSS variable here

**Review by krgarima — APPROVED** [review]
Add screenshots of the extension for review. Rest looks fine to me.

File: public/manifest.json (hunk 1 above)
Comment (TejasShekar) [reply public/manifest.json:2]: Yes. Initially it was v3. But mozilla add-on requires the manifest to be v2 for the extension to be published. So, made that change.

File: src/App.css (hunk 2 above)
Comment (TejasShekar) [reply src/App.css:14]: I'm using it only at one place. So, did not use variable here.

**Comment by TejasShekar** [comment]
> Add screenshots of the extension for review. Rest looks fine to me.

Preview Link has been added.

File: src/App.css [hunk 3]
```diff
@@ -72,7 +86,7 @@ p {
 
 .focus-edit {
   background: transparent;
-  color: gold;
+  color: var(--yellow);
```
Comment (SHIKHASINGH1506) [inline src/App.css:89]: name of the color variable could be better like warning.

**Review by SHIKHASINGH1506 — APPROVED** [review]
1. left suggestion regarding color variable name, minor one
2. rest looks good

**Pull request merged**